### PR TITLE
Added history page

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Added history page.

### CODE REVIEW

**Review by ark2002 — APPROVED** [review]
code looks good , context works right . good to merge

File: src/Pages/History/history.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    const deleteAllHistoryVideos = async (token) => {
+        await deleteAllFromHistoryService(token);
+        dispatchVideo({type : "DELETE_ALL_HISTORY_VIDEOS"})
+      }
+      
+      const deleteFromHistory = async (video , token) => {
+        await deleteFromHistoryService(video , token);
+        dispatchVideo({type : "DELETE_HISTORY_VIDEO" , payload : video})
+      }
+
```
Comment (viru2001) [inline src/Pages/History/history.js:53]: you can move this code to utils or services

**Review by viru2001 — APPROVED** [review]
gave small suggestion . everything else is perfect

File: src/Pages/History/history.js (hunk 1 above)
Comment (syed-shaida-hussain) [reply src/Pages/History/history.js:53]: Will extract all the utils at the end.

**Pull request merged**